lib/reports/oclc_registration.rb

```diff
@@ -8,30 +8,32 @@ module Reports
   # Runs a report for an organization, outputting all of their
   # shared print commitments in an OCLC Registration-like format.
   class OCLCRegistration
```

Comment:
Essentially 2 changes to this class in order to set it up as a parent to `Phase3OCLCRegistration`:
1) Turning `finder` into `@finder` and moving it from `run` to `initialize`.
2) Changing the way the output file is determined,